crates/matrix-sdk/tests/integration/client.rs

```diff
@@ -630,3 +631,29 @@ fn serialize_session() {
         })
     );
 }
+
+#[test]
+fn sanitize_server_name() {
```

Comment:
Should I create a new file for this? I tried adding it to `main.rs` but it didn't run.

Reply:
This should be in `main.rs`, are you sure it didn't run? It's in a different piece of the output then (unit tests, not integration tests).

Reply:
Hmmm ok, I'll give it another try 👀

Reply:
Ok yes, worked fine this time. Not sure what my issue was before :)

Reply:
Like so?

Reply:
Yes!